source/includes/mongodb-compatibility-table-ruby.rst

```diff
@@ -0,0 +1,203 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Ruby Driver
+     - MongoDB 4.4
```

Comment:
FYI: 3.13 will introduce support for MongoDB 4.4. We don't have an entry for it on the [releases page](https://jira.mongodb.org/projects/RUBY?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page&status=released-unreleased), but there is currently a fix version for the RC.

@ccho-mongodb: I assume you may not add a row for 3.13 until server GA, but please make a note to add an asterisk next to the checkmark in the MongoDB 4.4 column and a foot note that it will not support OCSP out of the gate. Unlike other drivers, Ruby will be following up with that 4.4-related feature in a subsequent minor version (likely 3.14).

@p-mongo @egiurleo: I think it makes sense to preemptively create fix versions for the stable release that we know will be tagged. I understand why we wouldn't necessarily want to do that for beta/RCs (which may not come to fruition), but having the final release in JIRA would let us set the a meaning description for it (e.g. "MongoDB 4.4 support"), as some other languages do.

Reply:
I created https://jira.mongodb.org/browse/RUBY/fixforversion/29644.